pkg/collector/corechecks/oracle-dbm/init.go

```diff
@@ -62,18 +62,16 @@ func (c *Check) init() error {
 
 	if c.config.ReportedHostname != "" {
 		c.dbHostname = c.config.ReportedHostname
-	} else {
-		if i.HostName.Valid {
-			c.dbHostname = i.HostName.String
-		} else {
-			// host_name is null on Oracle Autonomous Database
-			c.dbHostname = i.InstanceName
-		}
+	} else if i.HostName.Valid {
+		c.dbHostname = i.HostName.String
 	}
 	if i.HostName.Valid {
 		tags = append(tags, fmt.Sprintf("real_hostname:%s", i.HostName.String))
 	}
-	tags = append(tags, fmt.Sprintf("host:%s", c.dbHostname), fmt.Sprintf("oracle_version:%s", c.dbVersion))
+	if c.dbHostname != "" {
+		tags = append(tags, fmt.Sprintf("host:%s", c.dbHostname), fmt.Sprintf("db_server:%s", c.dbHostname))
```

Comment:
is there a difference between `db_server` and `real_hostname`?

Reply:
fixed not, thanks.